src/Features/CSharp/Portable/Snippets/CSharpIntMainSnippetProvider.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.LanguageService;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Snippets;
+using Microsoft.CodeAnalysis.Snippets.SnippetProviders;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Snippets
+{
+    [ExportSnippetProvider(nameof(ISnippetProvider), LanguageNames.CSharp), Shared]
```

Comment:
❓ Is this name correct?

Reply:
➡️ It looks like `nameof(ISnippetProvider)` goes to a location which is ignored, and it just happens that all of the values are wrong in the code base. The intended use of a name in exporting would be to use `nameof(CSharpIntMainSnippetProvider)` or something similar. However, since the value is ignored and consistent with all the other cases, we can keep it like this for now.

Reply:
📝 It would be acceptable to send a PR to remove the Name property from this export attribute.